pyronear/datasets/openfire.py

```diff
@@ -101,38 +116,55 @@ def _check_exists(self, train=True):
         else:
             return self._root.joinpath(self._processed, self.test_file).is_file()
 
-    def download(self, threads=None, valid_pct=None):
-        """Download the OpenFire data if it doesn't exist in processed_folder already.
+    def create_directories(self, max_files=None):
```

Comment:
default value of `max_files` differs from line 57, is it normal?

Reply:
Thanks for catching it. It should be None there as well, otherwise the last file would be skipped (it was a bug on the original implementation)

Reply:
Also, perhaps rename it to `num_samples` for a less confusing variable naming

Reply:
Done